packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendar.tsx

```diff
@@ -291,12 +295,13 @@ export const DatepickerCalendar: ComponentWithAs<'div', DatepickerCalendarProps>
                       createShorthand(DatepickerCalendarHeaderCell, calendarHeaderCell, {
                         defaultProps: () =>
                           getA11yProps('calendarHeaderCell', {
+                            'aria-label': days[(dayNumber + firstDayOfWeek) % DAYS_IN_WEEK],
                             content: shortDays[(dayNumber + firstDayOfWeek) % DAYS_IN_WEEK],
                             key: dayNumber,
                           }),
                       }),
                     )}
-                    {_.map(visibledGrid, week => renderWeekRow(week))}
+                    {_.map(visibledGrid, (week, idx) => renderWeekRow(week))}
```

Comment:
```suggestion
                    {_.map(visibledGrid, week => renderWeekRow(week))}
```
`idx` is not used, or am I missing something?